addons/account/wizard/account_report_common.py

```diff
@@ -45,4 +45,5 @@ def check_report(self):
         data['form'] = self.read(['date_from', 'date_to', 'journal_ids', 'target_move', 'company_id'])[0]
         used_context = self._build_contexts(data)
         data['form']['used_context'] = dict(used_context, lang=get_lang(self.env).code)
+        data['form']['res_company'] = data['form']['company_id']
```

Comment:
Is this working since `type(self.read(['company_id']))` is a `list`, not a recordset?